legacy.war throws an error when it is deployed

### Description

When legacy.war is deployed, these errors are thrown in the logs.

```
Error deploying web application archive [/var/lib/tomcat9/webapps/legacy.war]
java.lang.IllegalStateException: Error starting child
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'dataSource' defined in class path resource [org/springframework/boot/autoconfigure/jdbc/DataSourceConfiguration$Hikari.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantia>
Caused by: org.springframework.beans.BeanInstantiationException: Failed to instantiate [com.zaxxer.hikari.HikariDataSource]: Factory method 'dataSource' threw exception; nested exception is java.lang.IllegalStateException: Cannot load driver class: com.mysql.cj.jdbc.Driver
Caused by: java.lang.IllegalStateException: Cannot load driver class: com.mysql.cj.jdbc.Driver
Deployment of web application archive [/var/lib/tomcat9/webapps/legacy.war] has finished in [11,764] ms
```

--- a/scripts/afterInstall.sh
+++ b/scripts/afterInstall.sh
@@ -20,8 +20,8 @@ chown tomcat:tomcat $CATALINA_HOME/webapps/ROOT.war
 echo "Copying legacy.war to $BUILD_DIR"
 cp $BUILD_FILES/legacy.war $BUILD_DIR
 
-echo "Injecting application.properties into legacy.war"
-zip -g legacy.war WEB-INF/classes/application.properties
+echo "Injecting application-legacy.properties into legacy.war"
+zip -g legacy.war WEB-INF/classes/application-legacy.properties
 
 echo "Moving legacy.war to $CATALINA_HOME/webapps/legacy.war"
 mv legacy.war $CATALINA_HOME/webapps/legacy.war

--- a/scripts/afterInstall.sh
+++ b/scripts/afterInstall.sh
@@ -2,6 +2,7 @@
 
 export HOME=/home/ubuntu
 export BUILD_DIR=$HOME/build-folder
+export LEGACY_BUILD_DIR=$HOME/legacy-build-folder
 export BUILD_FILES=$HOME/wise-build-files
 export CATALINA_HOME=/var/lib/tomcat9
 
@@ -17,11 +18,14 @@ echo "Moving wise.war to $CATALINA_HOME/webapps/ROOT.war"
 mv wise.war $CATALINA_HOME/webapps/ROOT.war
 chown tomcat:tomcat $CATALINA_HOME/webapps/ROOT.war
 
-echo "Copying legacy.war to $BUILD_DIR"
-cp $BUILD_FILES/legacy.war $BUILD_DIR
+echo "Changing to $LEGACY_BUILD_DIR directory"
+cd $LEGACY_BUILD_DIR
+
+echo "Copying legacy.war to $LEGACY_BUILD_DIR"
+cp $BUILD_FILES/legacy.war $LEGACY_BUILD_DIR
 
 echo "Injecting application-legacy.properties into legacy.war"
-zip -g legacy.war WEB-INF/classes/application-legacy.properties
+zip -g legacy.war WEB-INF/classes/application.properties
 
 echo "Moving legacy.war to $CATALINA_HOME/webapps/legacy.war"
 mv legacy.war $CATALINA_HOME/webapps/legacy.war

--- a/scripts/beforeInstall.sh
+++ b/scripts/beforeInstall.sh
@@ -89,7 +89,7 @@ sudo -u ubuntu -g tomcat mkdir $HOME/googleTokens
 echo "Copying application.properties file to the build folder"
 cp $BUILD_FILES/api/$ENV/application.properties $BUILD_DIR/WEB-INF/classes/application.properties
 
-echo "Copying application-legacy.properties file to the build folder"
+echo "Copying application-legacy.properties file to the legacy build folder"
 cp $BUILD_FILES/api/$ENV/application-legacy.properties $LEGACY_BUILD_DIR/WEB-INF/classes/application.properties
 
 echo "Installing network drive package"

--- a/scripts/beforeInstall.sh
+++ b/scripts/beforeInstall.sh
@@ -87,6 +87,9 @@ sudo -u ubuntu -g tomcat mkdir $HOME/googleTokens
 echo "Copying application.properties file to the build folder"
 cp $BUILD_FILES/api/$ENV/application.properties $BUILD_DIR/WEB-INF/classes/application.properties
 
+echo "Copying application-legacy.properties file to the build folder"
+cp $BUILD_FILES/api/$ENV/application-legacy.properties $BUILD_DIR/WEB-INF/classes/application-legacy.properties
+
 echo "Installing network drive package"
 apt-get install nfs-common -y
 

--- a/scripts/beforeInstall.sh
+++ b/scripts/beforeInstall.sh
@@ -2,6 +2,7 @@
 
 export HOME=/home/ubuntu
 export BUILD_DIR=$HOME/build-folder
+export LEGACY_BUILD_DIR=$HOME/legacy-build-folder
 export BUILD_FILES=$HOME/wise-build-files
 export CATALINA_HOME=/var/lib/tomcat9
 
@@ -80,7 +81,8 @@ cp $BUILD_FILES/api/$ENV/wise.conf /etc/nginx/sites-enabled/wise.conf
 systemctl restart nginx
 
 echo "Creating additional folders for WISE"
-mkdir -p $HOME/build-folder/WEB-INF/classes
+mkdir -p $BUILD_DIR/WEB-INF/classes
+mkdir -p $LEGACY_BUILD_DIR/WEB-INF/classes
 sudo -u ubuntu -g ubuntu mkdir $HOME/backup
 sudo -u ubuntu -g tomcat mkdir $HOME/googleTokens
 
@@ -88,7 +90,7 @@ echo "Copying application.properties file to the build folder"
 cp $BUILD_FILES/api/$ENV/application.properties $BUILD_DIR/WEB-INF/classes/application.properties
 
 echo "Copying application-legacy.properties file to the build folder"
-cp $BUILD_FILES/api/$ENV/application-legacy.properties $BUILD_DIR/WEB-INF/classes/application-legacy.properties
+cp $BUILD_FILES/api/$ENV/application-legacy.properties $LEGACY_BUILD_DIR/WEB-INF/classes/application.properties
 
 echo "Installing network drive package"
 apt-get install nfs-common -y

